examples/wallet/program/src/main.rs

```diff
@@ -1,6 +1,5 @@
 //! The Spacemesh standard wallet template.
 #![no_main]
-#![no_std]
```

Comment:
We need to discuss the implications of this change

Reply:
It's not necessary. I thought it was added by mistake as other examples don't have it. Reverting.